cli-plugins/manager/hooks.go

```diff
@@ -38,20 +37,20 @@ func RunCLICommandHooks(ctx context.Context, dockerCli command.Cli, rootCmd, sub
 
 // RunPluginHooks is the entrypoint for the hooks execution flow
 // after a plugin command was just executed by the CLI.
-func RunPluginHooks(ctx context.Context, dockerCli command.Cli, rootCmd, subCommand *cobra.Command, args []string) {
+func RunPluginHooks(ctx context.Context, dockerCli ConfigProvider, rootCmd, subCommand *cobra.Command, args []string) {
 	commandName := strings.Join(args, " ")
 	flags := getNaiveFlags(args)
 
 	runHooks(ctx, dockerCli, rootCmd, subCommand, commandName, flags, "")
 }
 
-func runHooks(ctx context.Context, dockerCli command.Cli, rootCmd, subCommand *cobra.Command, invokedCommand string, flags map[string]string, cmdErrorMessage string) {
+func runHooks(ctx context.Context, dockerCli ConfigProvider, rootCmd, subCommand *cobra.Command, invokedCommand string, flags map[string]string, cmdErrorMessage string) {
 	nextSteps := invokeAndCollectHooks(ctx, dockerCli, rootCmd, subCommand, invokedCommand, flags, cmdErrorMessage)
 
-	hooks.PrintNextSteps(dockerCli.Err(), nextSteps)
+	hooks.PrintNextSteps(rootCmd.ErrOrStderr(), nextSteps)
```

Comment:
This is a difference; before this, we would take the `Err()` output from the CLI, and now we switched to the Cobra cmd's `StdErr()` - we should look in our code altogether to consider using that (instead of depending on the DockerCLI to provide us the stderr/stdout; I think they're coupled either way.